server/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const cors = require('cors')
+
+app.use(cors());
+app.use(express.json());
+
+const search = require('./data')
+
+
+app.get('/', (req, res) => {
+    res.json('It works!')
+})
+
+//Reading aspects of the code
+
+app.get('/search', (req, res) => {
+    res.json(search)
+})
+
+app.get('/search/:name', (req, res) => {
```

Comment:
Another maybe unnecessary suggestion considering the size of the file but could add a router for the search routes. Probably a waste of time considering there is literally only one route